packages/assets-controllers/src/AccountTrackerController.ts

```diff
@@ -128,29 +134,29 @@ export class AccountTrackerController extends StaticIntervalPollingControllerV1<
    * Creates an AccountTracker instance.
    *
    * @param options - The controller options.
-   * @param options.onPreferencesStateChange - Allows subscribing to preference controller state changes.
-   * @param options.getIdentities - Gets the identities from the Preferences store.
-   * @param options.getSelectedAddress - Gets the selected address from the Preferences store.
    * @param options.getMultiAccountBalancesEnabled - Gets the multi account balances enabled flag from the Preferences store.
    * @param options.getCurrentChainId - Gets the chain ID for the current network from the Network store.
    * @param options.getNetworkClientById - Gets the network client with the given id from the NetworkController.
+   * @param options.onSelectedAccountChange - A function that subscribes to selected account changes.
+   * @param options.getInternalAccounts - A function that returns the internal accounts.
+   * @param options.getSelectedAccount - A function that returns the selected account.
    * @param config - Initial options used to configure this controller.
    * @param state - Initial state to set on this controller.
    */
   constructor(
     {
-      onPreferencesStateChange,
-      getIdentities,
-      getSelectedAddress,
+      onSelectedAccountChange,
+      getInternalAccounts,
+      getSelectedAccount,
       getMultiAccountBalancesEnabled,
       getCurrentChainId,
       getNetworkClientById,
     }: {
-      onPreferencesStateChange: (
-        listener: (preferencesState: PreferencesState) => void,
+      onSelectedAccountChange: (
+        listener: (internalAccount: InternalAccount) => void,
       ) => void;
-      getIdentities: () => PreferencesState['identities'];
-      getSelectedAddress: () => PreferencesState['selectedAddress'];
+      getInternalAccounts: AccountsController['listAccounts'];
```

Comment:
```suggestion
      getInternalAccounts: getInternalAccounts;
```

We could use types we defined above?